e2e/tests/init_test.go

```diff
@@ -123,6 +123,85 @@ func (s *suite) TheUserShouldWaitForSeconds(dur int64) {
 	time.Sleep(time.Duration(dur) * time.Second)
 }
 
+func (s *suite) TheUserStakesAWithUpoktFromTheAccount(actorType string, amount int64, accName string) {
+	args := []string{
+		"tx",
+		actorType,
+		fmt.Sprintf("stake-%s", actorType),
+		fmt.Sprintf("%dupokt", amount),
+		"--from",
+		accName,
+		keyRingFlag,
+		"-y",
+	}
+	res, err := s.pocketd.RunCommandOnHost("", args...)
+	if err != nil {
+		s.Fatalf("error staking %s: %s", actorType, err)
+	}
+	s.pocketd.result = res
+}
+
+func (s *suite) TheUserUnstakesAFromTheAccount(actorType string, accName string) {
+	args := []string{
+		"tx",
+		actorType,
+		fmt.Sprintf("unstake-%s", actorType),
+		"--from",
+		accName,
+		keyRingFlag,
+		"-y",
+	}
+	res, err := s.pocketd.RunCommandOnHost("", args...)
+	if err != nil {
+		s.Fatalf("error unstaking %s: %s", actorType, err)
+	}
+	s.pocketd.result = res
+}
+
+func (s *suite) TheForAccountIsNotStaked(actorType, accName string) {
+	found, _ := s.getStakedAmount(actorType, accName)
+	if found {
+		s.Fatalf("account %s is staked", accName)
+	}
+}
+
+func (s *suite) TheForAccountIsStakedWithUpokt(actorType, accName string, amount int64) {
+	found, stakeAmount := s.getStakedAmount(actorType, accName)
+	if !found {
+		s.Fatalf("account %s is not staked", accName)
+	}
+	if int64(stakeAmount) != amount {
+		s.Fatalf("account %s stake amount is not %d", accName, amount)
+	}
+}
+
+func (s *suite) getStakedAmount(actorType, accName string) (bool, int) {
+	s.Helper()
+	args := []string{
+		"query",
+		actorType,
+		fmt.Sprintf("list-%s", actorType),
```

Comment:
Have you checked if using `show` instead of `list` is easier?

![Screenshot 2023-10-26 at 3 26 44 PM](https://github.com/pokt-network/poktroll/assets/1892194/66c7b092-8f4d-4ec6-8251-e1bc0698af79)

Reply:
So the reason I am not using `show-gateway` is because it errors if the key isn't found (aka not staked) which causes the test to fail. We could instead check the combined output for this error message but it overcomplicates what we are trying to do here.

Plus these tests should never result in a paginated result so I think this is fine.